src/main/java/codegym/tequila/fisioapp/aop/ControllersTimeLogAspect.java

```diff
@@ -0,0 +1,40 @@
+package codegym.tequila.fisioapp.aop;
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+@Aspect
+public class ControllersTimeLogAspect {
+
+    private static final Logger logger = LoggerFactory.getLogger(ControllersTimeLogAspect.class);
+
+    @Pointcut("within(codegym.tequila.fisioapp.controller.*)")
+    public void allControllersMethods() {}
+
+
+    @Around("allControllersMethods() || @annotation(codegym.tequila.fisioapp.aop.TimeLogAnotation)")
```

Comment:
Even when it wouldn't be the best practice, the controller could still have other methods that are not the ones we want measure, like private methods or so.

For a real project I'd recommend to remove the "allControllersMethod" condition here but for educational purposes it's great you included both and I prefer to keep it.